File: ts-sdk/src/sdk/models/shared/ledgerstorage.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";


export class LedgerStorage extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=driver" })
  driver: string;

  @SpeakeasyMetadata({ data: "json, name=ledgers" })
  ledgers: string[];
}

File: python-sdk/src/formanceapi/models/shared/ledgerstorage.py
import dataclasses
from dataclasses_json import dataclass_json
from formanceapi import utils


@dataclass_json
@dataclasses.dataclass
class LedgerStorage:
    driver: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('driver') }})
    ledgers: list[str] = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('ledgers') }})
    